torchvision/transforms/_core.py

```diff
@@ -23,17 +24,37 @@ def query_sample(sample: Any, fn: Callable[[Any], Optional[T]]) -> Iterator[T]:
             yield result
 
 
+class TransformDispatch():
+    def __init__(cls, transforms):
+        cls.transforms = transforms
+        cls.transforms.set_reset_auto(False)
```

Comment:
Not sure why you renamed `self` to `cls` here, since the value will be an actual instance rather than a class. If you want the class, you need to decorate the method with `@classmethod`. Plus, some dunder method are implicitly class methods such as `__init_subclass__`. 

For everything that I see below, you most likely do not want to use class methods and attributes as they will be the same for every object of that class.

Reply:
sorry about that it's a mistake from my part, will fix it